src/components/application_manager/include/application_manager/commands/mobile/subscribe_vehicle_data_request.h

```diff
@@ -113,6 +113,26 @@ class SubscribeVehicleDataRequest : public CommandRequestImpl {
       const smart_objects::SmartObject& msg_params) const;
 
   /**
+   * @brief Checks if current application and other applications
+   * were subscribed to VI, prepare data that need to send to mobile app
+   * or HMI.
+   * @param app contains application
+   * @param out_info contains resulting info for sending to mobile app
+   * @param out_result_code contains result code for sending to mobile app
+   * @param out_response_params contains parameters that SDL sends to
+   * mobile application
+   * @param out_request_params contains parameters that SDL sends to
+   * HMI
+   * @param result contains result that SDL sends to mobile app.
+   */
+  void CheckVISubscribtions(ApplicationSharedPtr app,
```

Comment:
@dtrunov 
I assume that result is also returned parameter, so please name it appropriately.